articles/sentinel/normalization-parsers-list.md

```diff
@@ -26,7 +26,7 @@ To use ASIM alert event parsers, deploy the parsers from the [Microsoft Sentinel
 | **Source** | **Notes** | **Parser**
 | --- | --------------------------- | ---------- |
 | **Defender XDR Alerts** | Microsoft Defender XDR alert events (in the `AlertEvidence` table). | `ASimAlertEventMicrosoftDefenderXDR` |
-| **Exchange 365 administrative events** | SentinelOne Singlularity `Threats.` events (in the `SentinelOne_CL` table). | `ASimAlertEventSentinelOneSingularity` |
+| **Exchange 365 administrative events** | SentinelOne Singularity `Threats.` events (in the `SentinelOne_CL` table). | `ASimAlertEventSentinelOneSingularity` |
```

Comment:
One additional change needed here. Please change **Exchange 365 administrative events** to **SentinelOne Singularit**

I missed noticing it previously while publishing.

Reply:
Other spelling corrections looking good.

Reply:
I can do that change, buts from my side at least, its not a spelling problem, its a content problem.

The fix was just that the system is called Singularity and no Singlularity as in the original form.

Reply:
If you can make the change, please go ahead, otherwise I'll have to do it in a separate PR.

Reply:
done